config/config.py.template

```diff
@@ -622,22 +612,19 @@ MLLIB_GMM_TEST_OPTS = MLLIB_COMMON_OPTS + [
 
 if MLLIB_SPARK_VERSION >= 1.3:
     MLLIB_TESTS += [("gmm", MLLIB_PERF_TEST_RUNNER, SCALE_FACTOR,
-        MLLIB_JAVA_OPTS, [ConstantOption("gmm")] + MLLIB_CLUSTERING_TEST_OPTS)]
+        MLLIB_JAVA_OPTS, [ConstantOption("gmm")] + MLLIB_GMM_TEST_OPTS)]
 
 MLLIB_LDA_TEST_OPTS = MLLIB_COMMON_OPTS + [
-    OptionSet("num-documents", [10000], can_scale=True),
-    OptionSet("num-vocab", [1000], can_scale=False),
+    OptionSet("num-documents", [50000], can_scale=True),
+    OptionSet("num-vocab", [10000], can_scale=False),
```

Comment:
They somewhere between the parameters in the original defaults and the ones I used for QA last cycle.  Not sure of the exact running time.